public/src/js/script.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * 
+ * Displays information about the method been called.
+ * @param {String} id - id of method to display
+ * @returns {void}
+ */
+const showMethodDetail = (id) => {
+  $('.details pre div').map((index, domElement) => {
+    domElement.style.display = 'none';
+    if(id) {
+      const attr = domElement.getAttribute('id');
+      if(id === attr) {
+        domElement.style.display = 'block';
+      }
+    }
+  });
+};
+
+/**
+ * Runs the method selected.
+ * @returns {void}
+ */
+const runMethod = () => {
+  let result = null;
+  const methodCalled = $('#string-method').val();
+  let value = $('#string-value').val(); 
+  if(methodCalled && value.length > 0) {
+    if(methodCalled in String.prototype) {
```

Comment:
Can't this condition be used in line 27 since if no `methodCalled` only checks if there is actually a value in `methodCalled` and if there is no value then `methodCalled in String.prototype === false`.